static/src/javascripts/projects/common/modules/identity/consent-journey.js

```diff
@@ -82,14 +84,20 @@ const submitJourneyAnyway = (buttonEl: HTMLElement): void => {
     });
 };
 
+const setLocalHasRepermissionedFlag = (): void => {
```

Comment:
When is this cookie actually set? I thought we were setting it when a user hits the repermission journey - in which case the naming of the cookie is a little misleading as it doesn't map to our internal `hasRepermissioned` flag.

Reply:
jon - it's set just as they arrive and your point makes a lot of sense, i'll rename it in a separate pr

Reply:
Thanks!